frontend/app/tests/contract/.eslintrc.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   rules: {
     'max-lines': 'off',
-    'import/max-dependencies': 'off'
-  }
+    'import/max-dependencies': 'off',
```

Comment:
btw if the PR is rebased on top of develop, we recently migrated to the new flat config model so this is not needed anymore, at least one of this rules should apply automatically now to `spec.ts`